gluetool/log.py

```diff
@@ -113,7 +113,7 @@ def warn_sentry(self, message, *args, **kwargs):
   File "{{ filepath }}", line {{ lineno }}, in {{ fn }}
     {{ text | default('') }}
 
-    Local variables:{% for name in frame.f_locals.iterkeys() | sort %}
+    Local variables:{% for name in frame.f_locals.items() | sort %}
```

Comment:
I'm kind of worried about this, as `iteritems` and `items` differ in Pythons 2 and 3, and `items` will materialize list of items when being run under Python 2. We might use something like this, maybe in a simple "workarounds" module or in `utils`, don't know:

```
import six

env = jinja2.Environment(loader=jinja2.FileSystemLoader('your_dir'))
env.globals['six_iteritems'] = six.iteritems
template = env.get_template('your_template')
```

We have similar stuff in `colors` already, to inject our `style` helper into Jinja templates as a filter.

Reply:
why are you worried? I guess list of locals can be long, but not so much usually. I do not mind adding a helper, I just do not see the value for this particular case. Am I missing something?

Reply:
It's just a matter of different behavior under Python2 and Python3. I'm not concerned about the performance issues, as you said, the list of locals won't be long enough to cause any problems. It will just behave differently when running on different Python versions which I'm not a big fan of.